core/templates/dev/head/domain/topic/topic-creation-backend-api.service.spec.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for TopicCreationBackendApiService.
+ */
+
+require('domain/topic/topic-creation-backend-api.service.ts');
```

Comment:
@bansalnitish, Are we expecting to have the "*backend-api.service.ts" inside domain folder?

Reply:
Aren't all backend api services in the domain folder?

Reply:
@aks681, Yeah, that's my concern whether we expect service files to be inside the domain folder?

/cc @bansalnitish

Reply:
I think yes we should pile up these services in domain folder.

Reply:
@bansalnitish I moved them into the domain folder